src/IO/exportUBC.jl

```diff
@@ -75,3 +75,13 @@ function exportUBCOcTreeModel(name::AbstractString, mesh::OcTreeMesh, model::Uni
     close(f)
     return
 end
+
+function outputOctreeMesh(name::AbstractString, mesh::OcTreeMesh)
```

Comment:
Removed this one - needs a better name and documentation.  Can be added back later.